usr/share/rear/layout/prepare/GNU/Linux/130_include_mount_subvolumes_code.sh

```diff
@@ -25,7 +28,7 @@ btrfs_subvolumes_setup() {
     mountopts="$3 $4"
     # Empty device or mountpoint may indicate an error. In this case be verbose and inform the user:
     if test -z "$device" -o -z "$mountpoint" ; then
-        LogPrint "Empty device='$device' or mountpoint='$mountpoint' may indicate an error, skipping btrfs_subvolumes_setup( $@ )."
+        LogPrintError "Empty device='$device' or mountpoint='$mountpoint' may indicate an error, skipping btrfs_subvolumes_setup( $@ )."
         Log "Return 0 from btrfs_subvolumes_setup( $@ )"
```

Comment:
It is my understanding that file system creation code as in `130_include_filesystem_code.sh` makes ReaR abort if any error occurs (and so does the new Btrfs setup implementation). Since subvolumes are file system-like objects (Btrfs doubles as a volume manager in this respect), it can be risky to continue with an incomplete setup: For example, the wrong disk partition may fill up with files restored from a backup.

Reply:
I do not want to change with this pull request how the old code behaves.
I don't know if I can error out here without causing subtle regressions.
I wished I had better descibed in my comment why I don't error out here.